### ISSUE TITLE
🐛 BUG: astro@next give 500 on fetch Content

### ISSUE THREAD

**Issue by hans-d** [snowpackjs/astro#1778]
### What package manager are you using?

npm

### What operating system are you using?

linux

### Describe the Bug

Trying to fetch (md) content, results in 
```
500 Internal Error
Unable to parse {"file":"/home/projects/github-csw8q2/src/pages/posts/[slug].astro","line":19,"column":39}
17 |        ],
18 |        "source": "\nWhoot!\n\n# some heading\n",
19 |        "html": "<p>Whoot!</p>\n<h1 id=\"some-heading\">some heading</h1>"
   |                                        ^
20 |      }
21 |    },
```

```
export async function getStaticPaths() {
    const data = Astro.fetchContent('../../data/posts/*.md')

    const allArticles = data.map(({ astro, file, url, ...article }) => {
        return {
      	    params: { slug: article.slug },
      	    props: {
	            article: article,
    		    content: astro.html,
	    	    md: astro.source,
	        }
        }
    })
    
    return allArticles

}
```

### Steps to Reproduce

1. `npm init astro` using template starter
2. add above function and a sample md
3. go to posts/first
4. Error! Describe what went wrong (and what was expected instead)...

It works with 0.20

### Link to Minimal Reproducible Example (Optional)

https://stackblitz.com/edit/github-csw8q2?file=src/pages/posts/[slug].astro

**Comment by hans-d** [comment]
also issues with `"astro": "^0.21.0-next.2"`

**Comment by natemoo-re** [comment]
Thanks, looks like we have some Markdown bugs still, will get these added to our checklist.

**Comment by hans-d** [comment]
not yet fixed on next.3

**Comment by natemoo-re** [comment]
Thanks for checking, @hans-d! `next.3` mostly has build-related fixes. We're looking into this one now and will add any updates to this issue.

**Comment by hans-d** [comment]
mostly checked this as I saw some markdown related issues also being included in next.3 ....

**Comment by rebelchris** [comment]
Same issue for post layouts that need the fetchContent (even in child components)

**Comment by rebelchris** [comment]
Wanted to confirm this is still open in `next.5`

**Comment by matthewp** [comment]
This bug seems related to the use of `<Debug/>` which is definitely a bug I'm not sure the one that you are expecting?

**Comment by hans-d** [comment]
I was also getting some issues without the Debug, but need to check if I can reproduce that.

**Comment by matthewp** [comment]
Thanks! This particular issue is fixed by #1882. If there is a unrelated issue to Debug please post a new issue (@rebelchris has seen things too)
